README.md

```diff
@@ -1,2 +1,13 @@
 # EigenData
 Contains Operator and AVS metadata
+
+## How to add your metadata
+* If you are an operator use [operators](/operators/) folder
+* If you are an avs use [avs](./avs/) folder
+
+### Steps
+1. Fork this repo.
+1. Create a folder with a unique name. This could be your organization or anything which identies your operator or avs.
```

Comment:
Suggestion: Create a folder within the operators or avs folders with a unique name